### PR TITLE
add fermi and bnl to cc

### PR DESCRIPTION
**Pull request by womullan**

### CODE REVIEW

**Review by ehneilsen — APPROVED** [review]
Looks good

**Review by drphilmarshall — APPROVED** [review]
Looks good to me. I'll chase down the BNL equivalent too, but this can be merged in the meantime I think. Thanks!

File: project_templates/technote_aastex/cookiecutter.json [hunk 1]
```diff
@@ -23,6 +23,7 @@
   "copyright_holder": [
     "Association of Universities for Research in Astronomy, Inc. (AURA)",
     "California Institute of Technology",
+    "Fermi Research Alliance, LLC",
```
Comment (drphilmarshall) [inline project_templates/technote_aastex/cookiecutter.json:26]:
```suggestion
    "Fermi Research Alliance, LLC",
    "Brookhaven Science Associates, LLC",
```

File: project_templates/technote_latex/cookiecutter.json [hunk 2]
```diff
@@ -37,6 +37,7 @@
   "copyright_holder": [
     "Association of Universities for Research in Astronomy, Inc. (AURA)",
     "California Institute of Technology",
+    "Fermi Research Alliance, LLC",
```
Comment (drphilmarshall) [inline project_templates/technote_latex/cookiecutter.json:40]:
```suggestion
    "Fermi Research Alliance, LLC",
    "Brookhaven Science Associates, LLC",
```

File: project_templates/technote_rst/cookiecutter.json [hunk 3]
```diff
@@ -33,6 +33,7 @@
   "copyright_holder": [
     "Association of Universities for Research in Astronomy, Inc. (AURA)",
     "California Institute of Technology",
+    "Fermi Research Alliance, LLC",
```
Comment (drphilmarshall) [inline project_templates/technote_rst/cookiecutter.json:36]:
```suggestion
    "Fermi Research Alliance, LLC",
    "Brookhaven Science Associates, LLC",
```

**Review by drphilmarshall — CHANGES_REQUESTED** [review]
Easy addition of BNL copyright statement to match Fermilab's.

File: project_templates/technote_latex/cookiecutter.json (hunk 2 above)
Comment (womullan) [reply project_templates/technote_latex/cookiecutter.json:41]: added alphabetically

**Comment by womullan** [comment]
@jonathansick is this ok?

**Review by jonathansick — APPROVED** [review]
Yep, this works.

**Pull request merged**